### PR TITLE
Add 'admin_only' decorator

### PR DESCRIPTION
**Pull request by iAliF**
Added 'admin_only' decorator to prevent duplicate coding in admin commands

### CODE REVIEW

**Comment by iAliF** [comment]
Hey. You're welcome. Thanks for your early response.

1. No. It'll be called only once in the decorator and won't be called in the function body.
2. Yes exactly. I agree with you. I'll change the default value of 'reply_message' to None. In this way, You'll pass the message if you need it. Otherwise, It'll be ignored.

File: tgbot/handlers/utils/decorators.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    def decorator(func: Callable):
+        @wraps(func)
+        def inner(update: Update, context: CallbackContext, *args, **kwargs):
+            user = User.get_user(update, context)
+
+            if not user.is_admin:
+                if reply_message is not None and update.effective_message:
+                    update.effective_message.reply_text(reply_message)
+                return
```
Comment (shamaevnn) [inline tgbot/handlers/utils/decorators.py:26]: There is unnecessary nesting level with if conditions, it might be reduced.

Comment (iAliF) [reply tgbot/handlers/utils/decorators.py:26]: Should I remove 'reply_message' option from the decorator?

Comment (shamaevnn) [reply tgbot/handlers/utils/decorators.py:26]: I think yes, in fact, it's not used because if user is not an admin, bot won't send anything.

Comment (iAliF) [reply tgbot/handlers/utils/decorators.py:26]: Yeah I agree with you. I just added it because it was existed before.

**Review by shamaevnn — APPROVED** [review]

**Comment by shamaevnn** [comment]
@ohld merge, if it looks good to you

**Review by ohld — APPROVED** [review]

**Pull request merged**